Azure/Azure-AppServices-Diagnostics-Portal
# Enable Diagnose and Solve to show Workflows

## Overview
This PR enables functionality in Diagnose and Solve to display workflows if a workflow is made public and is attached to a Category from the backend.


## Type of change




## Solution description
This PR enables functionality in Diagnose and Solve to display workflows if a workflow is made public and is attached to a Category from the backend.

## How Can This Be Tested? <span>&#128269;</span>
I have added a flag in generic-api-service called `fetchDetectorsAndWorkflows `. Whether workflows should be fetched or not is controlled by this flag. Currently it is false, which means it is the existing behavior and UI will only get detectors. For myself, I just set it to TRUE and I am able to test UI code easily against workflows. 

Once all the UI is updated to handle workflows, we can just update the Backend to return both Workflows and Detectors.

## Checklist <span>&#9989;</span>

## Screenshots After Fix (if appropriate):
![image](https://user-images.githubusercontent.com/5299838/231714772-535519e2-02d7-40ce-92d5-9c06b6a91381.png)
